### ISSUE TITLE
On Mac linuxkit now complains with "Unable to find qemu-system-x86_64" 

### ISSUE THREAD

**Issue by tristanls** [linuxkit/linuxkit#3525]
**Description**

Previously (v0.7), it seems that linuxkit bundled whatever was required to be able to build an AWS image on Mac. Now, it complains:
```
FATA[0000] Unable to find qemu-system-x86_64 within the $PATH
```

**Steps to reproduce the issue:**
```
$ cat demo.yaml
kernel:
  image: linuxkit/kernel:5.4.39
  cmdline: "console=ttyS0"
init:
  - linuxkit/init:v0.8
  - linuxkit/runc:v0.8
  - linuxkit/containerd:v0.8
  - linuxkit/ca-certificates:v0.8
  - linuxkit/memlogd:v0.8
onboot:
  - name: sysctl
    image: linuxkit/sysctl:v0.8
  - name: rngd
    image: linuxkit/rngd:v0.8
    command: ["/sbin/rngd", "-1"]
  - name: dhcpcd
    image: linuxkit/dhcpcd:v0.8
    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
services:
  - name: logwrite
    image: linuxkit/logwrite:v0.8
  - name: kmsg
    image: linuxkit/kmsg:v0.8
  - name: sshd
    image: linuxkit/sshd:v0.8
files:
  - path: etc/init.d/002-logging
    contents: |
      #!/bin/sh

      /usr/bin/memlogd -daemonize -max-line-len 12288
trust:
  org:
    - linuxkit
    - library
```
```
:~/code/go/src/github.com/linuxkit/linuxkit/src/cmd/linuxkit (master=)
$ git checkout v0.7
:~/code/go/src/github.com/linuxkit/linuxkit/src/cmd/linuxkit ((v0.7))
$ go install
:~/images
$ linuxkit build -format aws -size 256M -name demo demo.yaml
Extract kernel image: linuxkit/kernel:5.4.39
Add init containers:
Process init image: linuxkit/init:v0.8
Process init image: linuxkit/runc:v0.8
Process init image: linuxkit/containerd:v0.8
Process init image: linuxkit/ca-certificates:v0.8
Process init image: linuxkit/memlogd:v0.8
Add onboot containers:
  Create OCI config for linuxkit/sysctl:v0.8
  Create OCI config for linuxkit/rngd:v0.8
  Create OCI config for linuxkit/dhcpcd:v0.8
Add service containers:
  Create OCI config for linuxkit/logwrite:v0.8
  Create OCI config for linuxkit/kmsg:v0.8
  Create OCI config for linuxkit/sshd:v0.8
Add files:
  etc/init.d/002-logging
Create outputs:
  demo.raw
:~/code/go/src/github.com/linuxkit/linuxkit/src/cmd/linuxkit ((v0.7))
$ git checkout 319005e
:~/code/go/src/github.com/linuxkit/linuxkit/src/cmd/linuxkit ((319005e84...))
$ go install
:~/images
$ linuxkit build -format aws -size 256M -name demo demo.yaml
Extract kernel image: docker.io/linuxkit/kernel:5.4.39
Add init containers:
Process init image: docker.io/linuxkit/init:v0.8
Process init image: docker.io/linuxkit/runc:v0.8
Process init image: docker.io/linuxkit/containerd:v0.8
Process init image: docker.io/linuxkit/ca-certificates:v0.8
Process init image: docker.io/linuxkit/memlogd:v0.8
Add onboot containers:
  Create OCI config for linuxkit/sysctl:v0.8
  Create OCI config for linuxkit/rngd:v0.8
  Create OCI config for linuxkit/dhcpcd:v0.8
Add service containers:
  Create OCI config for linuxkit/logwrite:v0.8
  Create OCI config for linuxkit/kmsg:v0.8
  Create OCI config for linuxkit/sshd:v0.8
Add files:
  etc/init.d/002-logging
Create outputs:
  demo.raw
FATA[0000] Unable to find qemu-system-x86_64 within the $PATH 
FATA[0029] Error writing outputs: Error writing raw output: exit status 1 
```

**Comment by tristanls** [comment]
Tail of the verbose log output of the failure case:
```
...
INFO[0030] Create outputs:                              
DEBU[0030] format: [aws] demo                           
DEBU[0030] validating output: [aws]                     
INFO[0030]   demo.raw                                   
DEBU[0041] output linuxkit generated img: raw demo.raw size 256 
DEBU[0041] run /Users/tristan/code/go/bin/linuxkit: [-q run qemu -disk demo.raw,size=256M,format=raw -disk /Users/tristan/.moby/tmp/moby294145561/tardisk,format=raw -kernel /Users/tristan/.moby/linuxkit/mkimage-9ce95c2b6ebc4b58669d2b3a2edc9cdc6864954b47959373f0ab81e6d577b751] 
FATA[0000] Unable to find qemu-system-x86_64 within the $PATH 
FATA[0041] Error writing outputs: Error writing raw output: exit status 1 
```

**Comment by tristanls** [comment]
It looks like containerized QEMU has been removed in #3443. Feels like this should have made it into the [Removed section of the CHANGELOG](https://github.com/linuxkit/linuxkit/blob/master/CHANGELOG.md#removed).

**Comment by tristanls** [comment]
Resolved in scope of #3528
